bin/themis

```diff
@@ -4,22 +4,14 @@
 # Author : thinca <thinca+vim@gmail.com>
 # License: zlib License
 
-: "${THEMIS_VIM:=vim}"
-: "${THEMIS_ARGS:=-e -s}"
-
-if ! type realpath >/dev/null 2>&1; then
-	realpath() {
-		# NOTE: use X to avoid "XSMP opening connection" message.
-		"${THEMIS_VIM}" -u NONE -i NONE -N -X -V1 -e -s --cmd "echon resolve(argv(0))" --cmd "echo ''" --cmd quit "$1" 2>&1
-	}
+if [ -z "${THEMIS_HOME+x}" ]; then
+  THEMIS_HOME=$(cd "$(dirname "$0")/.." && pwd)
```

Comment:
This script must work with symbolic link([doc](https://github.com/thinca/vim-themis/blob/176868b6784d5b7281898c4cfb702d1da5f65b03/doc/themis.txt#L66)).  This doesn't work.